### PR TITLE
Update tools.json with Static Reviewer SAST tool

### PR DESCRIPTION
**Pull request by BlackTriumph993**
We will be pleased if you can add our SAST solution to your list. We are already in https://www.nist.gov/itl/ssd/software-quality-group/source-code-security-analyzers and we applied as CWE Effective Product.

### CODE REVIEW

File: _data/tools.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 [
+   {
+      "title": "Static Reviewer",
+      "url": "https://www.securityreviewer.net/",
+      "owner": "Security Reviewer",
+      "license": "Commercial",
+      "platforms": "Windows and Linux; on-Premises and in Cloud; Desktop, CLI and CI/CD & IDE plugin integration"
```
Comment (kingthorin) [inline _data/tools.json:7]:
```suggestion
      "platforms": "Windows and Linux; on-Premises and in Cloud; Desktop, CLI and CI/CD & IDE plugin integration",
```

File: _data/tools.json [hunk 2]
```diff
@@ -1,4 +1,13 @@
 [
+   {
+      "title": "Static Reviewer",
+      "url": "https://www.securityreviewer.net/",
+      "owner": "Security Reviewer",
+      "license": "Commercial",
+      "platforms": "Windows and Linux; on-Premises and in Cloud; Desktop, CLI and CI/CD & IDE plugin integration",
+      "note": "Static Reviewer executes code checks according most relevant Secure Coding Standards for 40+ programming languages, using built-in design 1000+ validation rules.",
```
Comment (kingthorin) [inline _data/tools.json:8]:
```suggestion
      "note": "Static Reviewer executes code checks according to the most relevant Secure Coding Standards for 40+ programming languages, using built-in design 1000+ validation rules.",
```

Seems like there’s a word missing after “design” as well, but I have no idea what you were going for… 🤷‍♂️

Comment (BlackTriumph993) [reply _data/tools.json:8]: "...using built-in 1000+ validation rules." the word "design" is a copy/paste error. Thx.

**Comment by kingthorin** [comment]
Let me know if you're good with the current content and I'll approve/merge.

**Review by kingthorin — APPROVED** [review]

**Pull request merged**